packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -440,10 +440,20 @@ class OpPerfTelemetry {
 }
 export interface IPerfSignalReport {
 	/**
-	 * Identifier for the signal being submitted in order to
+	 * Identifier to track broadcast signals being submitted in order to
 	 * allow collection of data around the roundtrip of signal messages.
 	 */
-	signalSequenceNumber: number;
+	broadcastSignalSequenceNumber: number;
+
+	/**
+	 * Number of broadcast signals sent during latency event window.
+	 */
+	broadcastSignalsSentDuringLatencyEventWindow: number;
+
+	/**
+	 * Counter for number of broadcast signals sent during latency event window.
+	 */
+	broadcastSignalCounterForLatencyEventWindow: number;
 
```

Comment:
Does feel odd to have to things that are "number of broadcast signals sent during latency event window". Can you add some more comments to distinguish the two uses?